generatebundlefile/helm.go

```diff
@@ -73,9 +73,6 @@ func UnTarHelmChart(chartRef, chartPath, dest string) error {
 		return fmt.Errorf("Empty input value given for UnTarHelmChart")
 	}
 	_, err := os.Stat(dest)
-	if err != nil {
-		return fmt.Errorf("Not valid File %w", err)
-	}
 	if os.IsNotExist(err) {
```

Comment:
shouldn't this just move below IsNotExist?

Reply:
yea in this case we want to create the file structure if it doesn't exist, but would prefer to keep it this way vs a general error since it's more descriptive on the error type.

Reply:
What happens when err != NotExist? That case is not handled here.

Reply:
In that case the folder directory structure we want already exists to unpack into, so we keep running the rest of the function.

Reply:
What should happen if err == permission denied for example?

Reply:
I get what you're saying I can add an error check here for non-expected errors.

Reply:
Added a check in [3c1884a](https://github.com/aws/eks-anywhere-packages/pull/247/commits/3c1884ae98014fa17b4fdd49666de409f5a2a347)